WORKSPACE

```diff
@@ -43,30 +43,19 @@ load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kotlin_repositories", "kt_reg
 kotlin_repositories()
 kt_register_toolchains()
 
-# rules_proto defines abstract rules for building Protocol Buffers.
-http_archive(
-    name = "rules_proto",
-    sha256 = "2490dca4f249b8a9a3ab07bd1ba6eca085aaf8e45a734af92aad0c42d9dc7aaf",
-    strip_prefix = "rules_proto-218ffa7dfa5408492dc86c01ee637614f8695c45",
-    urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/rules_proto/archive/218ffa7dfa5408492dc86c01ee637614f8695c45.tar.gz",
-        "https://github.com/bazelbuild/rules_proto/archive/218ffa7dfa5408492dc86c01ee637614f8695c45.tar.gz",
-    ],
-)
+'''
+The proto_compiler and proto_java_toolchain bindings load the protos rules needed for the model
+module while helping us avoid the unnecessary compilation of protoc.
+'''
 
-load("@rules_proto//proto:repositories.bzl", "rules_proto_dependencies", "rules_proto_toolchains")
-rules_proto_dependencies()
-rules_proto_toolchains()
+bind(
+    name = "proto_compiler",
+    actual = "//tools:protoc"
+)
 
-# rules_java defines rules for generating Java code from Protocol Buffers.
-http_archive(
-    name = "rules_java",
-    sha256 = "ccf00372878d141f7d5568cedc4c42ad4811ba367ea3e26bc7c43445bbc52895",
-    strip_prefix = "rules_java-d7bf804c8731edd232cb061cb2a9fe003a85d8ee",
-    urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/rules_java/archive/d7bf804c8731edd232cb061cb2a9fe003a85d8ee.tar.gz",
-        "https://github.com/bazelbuild/rules_java/archive/d7bf804c8731edd232cb061cb2a9fe003a85d8ee.tar.gz",
-    ],
+bind(
```

Comment:
Is this using the protobuf bundled with Bazel? Is this the recommended way to do this? Can you point here & in code to documentation that explains this?

Reply:
Sure - there aren't any in-depth explanations but I have linked documentation related to the bind() command, where this technique is used, and an issue discussing something similar

Reply:
Thanks for the documentation. It slightly confuses me, though. The Bazel issue you linked to (https://github.com/bazelbuild/bazel/issues/7095) does discuss this as a problem, but the Bazel documentation actually suggests against using bind and that earlier issue doesn't mention bind as a solution.

Reply:
Yes, I couldn't find anything directly explaining this solution. I just have the example displayed here: https://github.com/google/startup-os/blob/master/WORKSPACE#L173 and found the thread I linked. In terms of the documentation for bind(), I figured it could be helpful if developers want to look into what the function does? But I guess they could also just google it?

Reply:
okay!